main.js

```diff
@@ -1,42 +1,56 @@
-// Note: This only seems to work in Repl.it for some reason but it does "kind of" work.
 
-
-
-
-
-  var towers = [[3, 2, 1], [], []];
-
-  var count = 0;
-
-  var moveTowers = function(start,end) {
-    var startPeg = start - 1; // so user doesn't have to ask for 0 when they mean 1
-    var endPeg = end - 1;     // so user doesn't have to ask for 0 when they mean 1
-    var emptyEndPeg = towers[endPeg].length === 0; // checks if peg is empty. if it's empty, merging is easier.
-    var notEmptyStartPeg = towers[startPeg].length != 0; // you don't want to waste a move moving an empty start peg into a peg that has value.
-    var lastItemInStartArray = towers[startPeg][towers[startPeg].length - 1]; // Last Item In Start Array
-    var lastItemInEndArray = towers[endPeg][towers[endPeg].length-1]; // Last Item in End Array
-    if (lastItemInStartArray < lastItemInEndArray || emptyEndPeg && notEmptyStartPeg) {
-      towers[endPeg].push(lastItemInStartArray);
-      towers[startPeg].pop();
-      count++;
-    } else {
-      console.log("Move is not legal.");
-    }
-
-    console.log(towers);
-    console.log('Move Count: ' + count);
-    checkWinner();
+let towers = [[3, 2, 1], [], []];
+let count = 0;
+
+moveTowers = (start, end) => {
+  const startPeg = start - 1
+  const endPeg = end - 1
+  const emptyEndPeg = towers[endPeg].length === 0 
+  const notEmptyStartPeg = towers[startPeg].length !== 0 
+  const lastItemInStartArray = towers[startPeg][towers[startPeg].length - 1]
+  const lastItemInEndArray = towers[endPeg][towers[endPeg].length-1]
+  if (lastItemInStartArray < lastItemInEndArray || emptyEndPeg && notEmptyStartPeg) {
+    towers[endPeg].push(lastItemInStartArray);
+    towers[startPeg].pop();
+    count++;
+  } else {
+    console.log("Move is not legal.");
+  }
+  printBoard()
+  checkWinner()
+}
+
+printBoard = () => {
+  const pegs = '---'
+  let i = 1
+  const pegsArrayMap = towers.map(x => i++ + ' ' + pegs + ' ' + x);
+  pegsArrayMap.forEach(function(peg) {
+    console.log(peg);
+  }); 
+  i = 1
+  moveCounter()
+}
+
+checkWinner = () => {
+  if ((towers[1].length === 3) || (towers[2].length === 3)) {
+    console.log('Congratulations, you won! It took you ', count, ' total moves.')
```

Comment:
Personally, I think I'd prefer to leverage [interpolatio](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Expression_interpolation) instead of concatenation, but that's probably more of a personal preference.

Reply:
taken care of while addressing [this](https://github.com/jpark799/towers-of-hanoi/pull/1#discussion_r320212659) comment